2200070/신용카드1.py

```diff
@@ -0,0 +1,32 @@
+T=int(input())
+# T=1
+
+for case in range(1, T+1):
+    c=list(map(int, input().split()))
+    f=[]
+    s=[]
+    result=0
+
+    for i in range(len(c)):
+        v=c[i]
+        if i == 0:
+            f.append(v)
+        elif i % 2 == 1:
+            s.append(v)
+        else:
+            f.append(v)
+
+    for n in f:
+        n1 = n * 2
+        result += n1
```

Comment:
좋은 풀이네요 👍 
```suggestion
    result = 2 * sum(f)
```
`sum` 함수를 이용해 나타내는 방법도 있습니다! 😄